src/test/java/com/epam/service/UserServiceImplTest.java

```diff
@@ -0,0 +1,30 @@
+package com.epam.service;
```

Comment:
Why this file like a new in pull request if it already exists in dev?

Reply:
Done